generators/getMetricMultipleSlugsGenerator.js

```diff
@@ -0,0 +1,179 @@
+const { metricsList } = require('./getMetricFunctions')
+
+const CURRENCY_DESCRIPTION = `
+* @param {string} currency The metric is calculated using a currency of choice.
+* Available currencies: (currencies)`
```

Comment:
`Available currencies: (currencies)` how does this work? Is `currencies` replaced by values?